config/main.py

```diff
@@ -519,7 +519,100 @@ def is_ipaddress(val):
         return False
     return True
 
+# class for locking entire config process
+class configdblock():
```

Comment:
Suggest rename class using Pascal case, which will also help readability: `ConfigDbLock`

Reply:
Sure, I will change the name, it shd be pascal case only right not camelcase. Also plz let me know if any other changes are needed before I raise 2nd commit. Thx.